CHANGELOG.md

```diff
@@ -17,9 +17,9 @@ Unreleased
 ----------
 ### Added
 * Add `terra-i18n` to the list provided to the duplicate package checker plugin in the webpack configuration.
-
-### Fixed
-* Allow custom mismatch tolerance of 0
+* Added prefix to theme-aggregator
+### Changed
```

Comment:
Oops, you will want to keep the existing entry and add your entry to the top of the CHANGELOG under the [`UNRELEASED` section](https://github.com/cerner/terra-toolkit/pull/367/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR10)